### PR TITLE
feat(HttpClient): Add Prefix And HttpClientCallerBase adds the ConfigureHttpClient method, which makes it more convenient to set timeouts and other configurations

### PR DESCRIPTION
**Pull request by zhenlei520**
### Docs
* Dapr.HttpClient: Add HttpClient ReadMe

### Features
* Caller.Core: Add GetStringAsync, GetByteArrayAsync, GetStreamAsync method overloads
* Dapr.HttpClient: Add Prefix And HttpClientCallerBase adds the ConfigureHttpClient method, which makes it more convenient to set timeouts and other configurations

### Fix
* Dapr.HttpClient: `methodName` support absolute address and relative address, support contains /

### CODE REVIEW

**Review by doddgu — CHANGES_REQUESTED** [review]

File: src/Caller/Masa.Utils.Caller.HttpClient/README.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+1. Modify `Program.cs`
+
+    ```` C#
+    builder.Services.AddCaller();
+    ````
+
+2. Add a new class `UserServiceCaller`
+
+    ```` C#
+    public class UserCaller: HttpClientCallerBase
```
[the remark this answers is not in the document]
Comment (zhenlei520) [reply src/Caller/Masa.Utils.Caller.HttpClient/README.md:84]: processed

File: src/Caller/Masa.Utils.Caller.HttpClient/README.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    {
+        protected override string BaseAddress { get; set; } = "http://localhost:5000";
+
+        public HttpCaller(IServiceProvider serviceProvider) : base(serviceProvider)
+        {
+        }
+
+        public Task<string> GetHello1Async() => CallerProvider.GetStringAsync("/Check/Hello1");
```
[the remark this answers is not in the document]
Comment (zhenlei520) [reply src/Caller/Masa.Utils.Caller.HttpClient/README.md]: processed

File: src/Caller/Masa.Utils.Caller.HttpClient/README.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+            httpClient.Timeout = TimeSpan.FromSeconds(5);
+        }
+    }
+    ````
+
+3. How to use UserCaller
+
+    ```` C#
+    app.MapGet("/Test/User/Check/Healthy", ([FromServices] UserCaller userCaller)
+        => userCaller.GetAsync<string>("/Check/Healthy"));
```
[the remark this answers is not in the document]
Comment (zhenlei520) [reply src/Caller/Masa.Utils.Caller.HttpClient/README.md]: processed

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**